crates/biome_service/src/diagnostics.rs

```diff
@@ -186,9 +174,6 @@ pub struct DirtyWorkspace;
         description = "The report can't be serialized, here's why: {reason}"
     )
 )]
```

Comment:
You forgot to remove the directives

Reply:
Hi @ematipico . Appolgize for it; missed it due to oversight. I've made the change.